campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -293,7 +293,18 @@
     var snippet = document.getElementById("mofo-eoy-fundraiser");
 
     snippet.addEventListener("show_snippet", function () {
-      setTimeout( function() {
```

Comment:
Interestingly enough where was no timeout specified, so the code got (almost) instantly executed. I 've tested this and it works.